src/Map/src/Exception/Exception.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
```suggestion
```

(and all the other ones 😅 )

Reply:
Yep!

This would be super useful if PHP-CS-Fixer could remove them automatically tho